docs/customizing_deis/logger_settings.rst

```diff
@@ -28,7 +28,18 @@ setting                                  description
 
 Settings used by logger
 -------------------------
-The logger component uses no keys from etcd.
+The following etcd keys are used by the logger component.
+
+====================================      ================================================================================
+setting                                   description
+====================================      ================================================================================
+/deis/logs/storageAdapterType             Type of storage adapter to use: ``file`` or ``memory``; if not set, ``file`` is assumed.  It is also possible so specify the size of the in-memory adapter's internal ring buffer using a value like: ``memory:<size>``.  1000 is the default size.
```

Comment:
It's lines.  Max line size is 65k.  You're right that this requires clarification.